### PR TITLE
README: add build method when git not installed

### PR DESCRIPTION
**Pull request by SleepyRoy**
This PR adds build method for people without git installed on computer.
I believe there're also many non-professionals or programmers unfamiliar with github (so they probably do not have git) but willing to try V.
This PR is just a start for this. I tested on windows and it worked.

### CODE REVIEW

**Comment by spytheman** [comment]
I am against merging this.

People that do not have git installed, should not try v at this stage, or should use the prebuilt releases instead. 

Adding instructions for how to manually patch stuff, so that it works without git is not helpful at all, neither to the non-programmers, nor for the maintainers, who then would have to support another case of slightly buggy installations (and which is worse, slightly different buggy installations for each new user...).

File: README.md [hunk 1]
```diff
@@ -60,6 +62,17 @@ V is being constantly updated. To update V, simply run:
 v up
 ```
 
+2. If you do not have Git installed
```
Comment (spytheman) [inline README.md]: The instructions for this case should be how to install git, NOT how to patch manually source files!

Comment (spytheman) [reply README.md]: Another option is to point people to the prebuilt releases (which are older, but are built specifically for people that do not want to track the latest and greatest).

**Comment by medvednikov** [comment]
I agree with @spytheman

**Comment by SleepyRoy** [comment]
@spytheman Yeah, it's just a workaround and I didn't think about the potential problems caused by the patch.

**Comment by SleepyRoy** [comment]
Close it.

**Pull request closed without merging**